MekHQ/src/mekhq/gui/StratconTab.java

```diff
@@ -394,6 +409,13 @@ private void repopulateTrackList() {
             stratconPanel.setVisible(false);
         }
     }
+    
+    private void showCampaignStateManagement(ActionEvent e) {
+        TrackDropdownItem selectedTrack = (TrackDropdownItem) cboCurrentTrack.getSelectedItem();
+        cmd.display(selectedTrack.contract.getStratconCampaignState(), selectedTrack.track, getCampaign().isGM());
+        cmd.setModalityType(ModalityType.APPLICATION_MODAL);
+        cmd.setVisible(true);
+    }
```

Comment:
This is short enough that it can just be made inline, and the modality doesn't make any sense to me outside of needing to consider the current GM state.

Reply:
I expect there will be more stuff that needs to be GMed as time goes on, and I don't like being able to click somewhere else and have the dialog disappear on me.